segmentation/include/pcl/segmentation/extract_clusters.h

```diff
@@ -202,6 +246,7 @@ namespace pcl
       return;
     }
 
+    eps_angle = std::max(std::abs(eps_angle), M_PI);
```

Comment:
This looks suspicious, eps usually represents a very small quantity and here you're taking the max of something which the name implies it should be small and M_PI which is large. Basically it will always be M_PI.

Edit: I just read the function signature and realized that the issue is in this variables name.

Reply:
It should be `std::min()`. Should I change it to min or is there something else needed to be done.

Reply:
Both readers have come to a different conclusion :fearful:

Reply:
> It should be `std::min()`. Should I change it to min or is there something else needed to be done.

Indeed you should change to min. eps is supposed to be a maximum threshold (very poor choice of name)

Reply:
It's an internal variable. Go ahead with a rename to `max_angle` or `upper_bound_angle` or something similar

Reply:
Changed to `min()` and renamed to `max_angle`